contracts/mainnet/connectors/euler/euler-rewards/events.sol

```diff
@@ -0,0 +1,13 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+contract Events {
+    event LogClaimed(
+        address user,
+        address token,
+        uint256 amt,
+        bytes32[] proof,
```

Comment:
@shriyatyagii I think we don't need to emit proofs.

Reply:
Removed